packages/scene-composer/src/components/three-fiber/EditorCamera.tsx

```diff
@@ -58,6 +58,7 @@ export const EditorMainCamera = forwardRef<Camera>((_, forwardedRef) => {
       return;
     }
     // Otherwise, return CameraControls for the current cameraControlsType. Do nothing for Immersive
+    console.log('Camera control type changed to: ', cameraControlsType);
```

Comment:
Was it intentional to leave the console log here?

Reply:
removed